README.md

```diff
@@ -117,26 +133,35 @@ SET INCLUDE=%PATH_TO_C2S_INSTALL_FOLDER%\include;%INCLUDE%
 SET CPATH=%PATH_TO_C2S_INSTALL_FOLDER%\include;%CPATH%
 ```
 
-### Test SYCLomatic
-#### Run in-tree LIT tests
+### Verify the Installation
+Before running SYCLomatic, it is important to verify that dpct/c2s(SYCLomatic binary) can be found in terminal.
+
+To do this, you need run the following command and get correct output.
```

Comment:
Before running SYCLomatic, it is important to verify that dpct/c2s(SYCLomatic binary) is installed correctly and PATH is set properly.
To do this, try running the following command and checking if dpct is available after the installation.